src/app/components/annotations/Annotation.js

```diff
@@ -311,28 +306,29 @@ class Annotation extends Component {
         }
       } else if (object.annotation_type === 'report_design') {
         const reportDesignChange = safelyParseJSON(activity.object_changes_json).data;
-        let reportState = this.props.intl.formatMessage(messages.editedBy);
         if (reportDesignChange && reportDesignChange[0]) {
-          reportState = reportDesignChange[1].state;
+          let reportState = '';
           if (reportDesignChange[0].state === reportDesignChange[1].state) {
-            reportState = this.props.intl.formatMessage(messages.editedBy);
+            reportState = '';
           } else if (reportDesignChange[1].state === 'published') {
             reportState = this.props.intl.formatMessage(messages.publishedBy);
-          } else {
+          } else if (reportDesignChange[1].state === 'paused') {
             reportState = this.props.intl.formatMessage(messages.pausedBy);
           }
+          if (reportState) {
+            contentTemplate = (
+              <FormattedMessage
+                id="annotation.reportDesignState"
+                defaultMessage="Fact-check {state} {author}"
+                description="Log entry indicating a report state has changed. Example: Fact-check [paused by|published by] author"
+                values={{
+                  state: reportState,
```

Comment:
Sawy, can you please refactor this as 2 full sentences where `author` is the only variable?

"Fact-check paused by {author}" and "Fact-check published by {author}" ?

Reply:
Done